utils/nctl/sh/scenarios-upgrades/upgrade_scenario_03.sh

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------
+# Synopsis.
+# ----------------------------------------------------------------
+
+# 1. Start v1 running at current mainnet commit.
+# 2. Execute some deploys to populate global state a little
+# 3. Upgrade all running nodes to v2
+# 4. Assert v2 nodes run & the chain advances (new blocks are generated)
+# 5. Start passive nodes.  The launcher should cause the v1 node to run, it should exit and the v2 node should then catch up.
+# 6. Assert passive nodes are now running.
```

Comment:
The original ticket mentions bonding/bidding/auction actions across upgrade points but this description seems to be outdated. Can you update it to match the test?

Reply:
Ninja emoji here